cli/standalone/binary.rs

```diff
@@ -134,7 +134,7 @@ pub enum NodeModules {
 pub struct Metadata {
   pub argv: Vec<String>,
   pub seed: Option<u64>,
-  pub permissions: PermissionsOptions,
+  pub permissions: PermissionFlags,
```

Comment:
Fix is here. Instead of storing the resolved `PermissionsOptions` we store the raw flags.